.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -52,7 +52,7 @@
     #- DEPLOYER_CLIENT_VERSION: "4.9.58"
     #- DEPLOYER_CLIENT_VERSION: "4.10.55"
     #- DEPLOYER_CLIENT_VERSION: "4.11.33"
-    - DEPLOYER_CLIENT_VERSION: "4.12.14"
+    - DEPLOYER_CLIENT_VERSION: "4.13.4"
```

Comment:
I don't like this duplication but we also maintain the latest version in `hack/deployer/config/plans.yml`.

Reply:
> We have identified an issue that caused pull requests to not reflect additional git pushes over the last several hours. New pushes will now be processed as normal.

The commit that changed the plans.yml should show up now on GH 🤔